### ISSUE TITLE
Deleted binaries running under /dev not collected

### ISSUE THREAD

**Issue by halpomeranz** [tclahr/uac#65]
The current code in artifacts/live_response/process/deleted.yaml ignores deleted executables that are installed under /dev ('loop_command: ... | grep -v -E "> /dev/|> /proc/" | ...'). So, for example, deleted EXEs running under /dev/shm are ignored:

```
# ls -l /proc/[0-9]*/exe 2>/dev/null | grep -E "\(deleted\)"
lrwxrwxrwx. 1 postfix postfix 0 Jul  1 21:20 /proc/58719/exe -> /dev/shm/.rk/lsof (deleted)
lrwxrwxrwx. 1 postfix postfix 0 Jul  1 21:20 /proc/58725/exe -> /dev/shm/.rk/xterm (deleted)
```

We would certainly like to collect the above files! 

I'm not sure what the reason was for excluding /dev, but maybe more granular logic is needed here?

**Comment by tclahr** [comment]
Yes, I will remove /dev from the binary collection, but I will keep it when UAC collects deleted file descriptors. I have seen some deleted file descriptors that point to sockets or pipes, or even devices.

**Comment by tclahr** [comment]
Done!
Merged into develop branch #76

**Comment by tclahr** [comment]
added a new code to collect copies of '/proc/[pid]/fd/*' from deleted processes even if they (file descriptors) are not shown up as being (deleted).
